linux/gcp-config.yaml

```diff
@@ -0,0 +1,161 @@
+receivers:
+  # scrape all metrics at 1m interval
+  hostmetrics:
+    collection_interval: 11s # for testing
+    scrapers:
+      cpu:
+        report_per_cpu: true
+      load:
+      memory:
+      disk:
+      filesystem:
+      network:
+      process:
+  # add receivers from Nic to extend the ability for MySQL, JVM and Apache
```

Comment:
In that doc I think I implied we'd have a separate config file for each packaging method, but I've since realized that's probably not necessary. We just need a separate config for each OS I think (i.e. one for Linux & one for Windows).

That makes me think:

- Remove Dockerfile and just use existing one
- Move `generate_tar.sh` into a "tar" directory
- Create a config directory, put this in there, and rename to maybe "config-gce-linux.yaml"?

Reply:
Also note this config file is quite outdated now - see https://github.com/GoogleCloudPlatform/opentelemetry-operations-collector/pull/2